index.js

```diff
@@ -87,12 +87,13 @@ exports.getTweetsOnThisDay = (req, res) => {
             // uses our app at 00:30 Feb 16, they'll get tweets made on Feb 15.
             // We need to change our date query to reflect the day wherever they are
             let offset = req.query.utcOffset;
-            let today = new Date();
+            let date = req.query.date || new Date();
```

Comment:
You'll need to merge master into your branch so this file can be updated.

Reply:
Ok. Done